sycl/include/CL/sycl/detail/pi.h

```diff
@@ -855,6 +855,17 @@ pi_result piEnqueueMemUnmap(
   const pi_event * event_wait_list,
   pi_event *       event);
 
+pi_result piGetDeviceFunctionPointer(
```

Comment:
rename this to piextDeviceGetFunctionPointer, and move to where other device interfaces are

Reply:
Done